From f90000cd4a998d923c9c90aaca57f6ae8cd5faae Mon Sep 17 00:00:00 2001
From: Richard Su <rwsu@redhat.com>
Date: Mon, 10 Feb 2014 16:52:26 -0800
Subject: [PATCH] Remove rabbitmq-server from boot-stack's element-deps

It is being removed to allow selection of other message brokers.

A parallel patch for tripleo-incubator will select rabbitmq-server
as the default in diskimage-builder image builds.

Change-Id: I530f2e40008d2a2d0dd6510116331f68e018a9be
---
 elements/boot-stack/element-deps | 1 -
 1 file changed, 1 deletion(-)

diff --git a/elements/boot-stack/element-deps b/elements/boot-stack/element-deps
index 5c2d678..e531acd 100644
--- a/elements/boot-stack/element-deps
+++ b/elements/boot-stack/element-deps
@@ -10,4 +10,3 @@ os-apply-config
 os-refresh-config
 neutron-openvswitch-agent
 neutron-server
-rabbitmq-server
-- 
1.8.5.3

